src/platform/mt8195/include/platform/lib/memory.h

```diff
@@ -20,7 +20,7 @@
 /* BOOT_WITH_DRAM ONLY */
 /* physical DSP addresses */
 #define DRAM_BASE 0x60000000
-#define DRAM_AUDIO_SHARED_SIZE 0x80000
+#define DRAM_AUDIO_SHARED_SIZE 0x280000
```

Comment:
wondering - is there a specific reason to use 0x280000? Not a round number like 0x200000 or 0x400000 or at least 0x300000?

Reply:
@lyakh  This is share buffer we define for audio.  We use the same buffer for legacy and sof audio. For legacy audio we reserve 0x80000 for each, currently mt8195 support 5 audio interface (DL2/DL3/DL10/UL4/UL5, DL10 is legacy audio), so we define it as 0x280000(although sof audio may not need 0x80000 but we keep flexibility for it.). Thanks.